packages/xo-server-test/sample.config.toml

```diff
@@ -3,6 +3,13 @@
   email = ''
   password = ''
 
+[servers]
+  [servers.default]
+    username = ''
+    password = ''
+    host = ''
+    allowUnauthorized = true
```

Comment:
This value can be modified by the user?

Reply:
No, you're right.

Reply:
Do you have an idea ? Because I remove this value and I add it to the params when I call `createTempServer()` and it returns: `this properties can only changed by an administrator`